Scripts/RemediationScripts/Remediate-EnableEncryptionInTransitForStorageAccounts.ps1

```diff
@@ -0,0 +1,691 @@
+          
+        .PARAMETER FilePath
+        Specifies the path to the file to be used as input for the roll back.
+       
+        .INPUTS
+        None. You cannot pipe objects to Disable-StorageEncryptionInTransit.
+       
+        .OUTPUTS
+        None. Disable-StorageEncryptionInTransit does not return anything that can be piped and used as an input to another command.
+       
+        .EXAMPLE
+        Disable-StorageEncryptionInTransit -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -FilePath  C:\Users\Documents\AzTS\Remediation\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\20211013_0608\EnableSecureTransit\StorageWithDisableHTTPS.csv
+
+        .LINK
+        None
+    #>
+
+    param (
+        [string]
+        [Parameter(Mandatory = $true, HelpMessage="Enter subscription id to perform rollback operation")]
+        $SubscriptionId,
+
+        [Switch]
+        [Parameter(Mandatory = $false , HelpMessage="Specifies validation of prerequisites for the command")]
+        $PerformPreReqCheck,
+
+        [switch]
+        [Parameter(Mandatory = $false, HelpMessage="Specifies a forceful remediation without any prompts")]
+        $Force,
+    
+        [string]
+        [Parameter(Mandatory = $true, HelpMessage="CSV file path which contain logs generated by remediation script to rollback remediation changes")]
+        $FilePath
+    )
+
+    Write-Host $([Constants]::DoubleDashLine)
+    Write-Host "Starting rollback operation to disable secure transfer on Storage Account(s) from subscription [$($SubscriptionId)]...." -ForegroundColor $([Constants]::MessageType.Info)
+    write-host $([Constants]::DoubleDashLine)
+     
+    Write-Host "[step 1 of 4]: Checking for prerequisites." 
+
+    if ($PerformPreReqCheck)
+    {
+        try
+        {
+            Write-Host "Setting up prerequisites..."
+            Setup-Prerequisites
+        }
+        catch
+        {
+            Write-Host "Error occurred while setting up prerequisites. Error: $($_)" -ForegroundColor $([Constants]::MessageType.Error)
+            break
+        }
+    }
+        
+    # Connect to AzAccount
+    $isContextSet = Get-AzContext
+    if ([string]::IsNullOrEmpty($isContextSet))
+    {       
+        Write-Host "Connecting to AzAccount..."
+        Connect-AzAccount -ErrorAction Stop
+        Write-Host "Connected to AzAccount" -ForegroundColor $([Constants]::MessageType.Update)
+    }
+
+    # Setting context for current subscription.
+    $currentSub = Set-AzContext -SubscriptionId $SubscriptionId -ErrorAction Stop -Force
+    Write-Host "Metadata Details: `nSubscriptionId: $($SubscriptionId) `nAccountName: $($currentSub.Account.Id) `nAccountType: $($currentSub.Account.Type)" 
+    Write-Host $([Constants]::SingleDashLine) 
+    Write-Host "Starting with subscription [$($SubscriptionId)]..." 
+    write-host $([Constants]::DoubleDashLine)
+
+    write-host "[step 2 of 4]: Validating the Account type."
```

Comment:
`write-host` to be replaced with `Write-Host`.

Reply:
done